tests/compat-in-docker.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -eu
+
+# compat-in-docker.sh
+#
+# This file is part of mbed TLS (https://tls.mbed.org)
+#
+# Copyright (c) 2019, ARM Limited, All Rights Reserved
```

Comment:
We should use our standard license header here, and in other new files, being sure to mention Apache 2.

See https://github.com/ARMmbed/mbedtls/blob/development/include/mbedtls/aes.h#L23-L38

Reply:
Done (I think) in commit 31f0081